filebeat/harvester/registry.go

```diff
@@ -0,0 +1,69 @@
+package harvester
+
+import (
+	"sync"
+
+	uuid "github.com/satori/go.uuid"
+)
+
+type Registry struct {
```

Comment:
**[golint]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
exported type Registry should have comment or be unexported